ci/playbooks/pre-doc.yml

```diff
@@ -23,5 +23,18 @@
         name:
           - make
           - python3
-          - golang
```

Comment:
```suggestion
    - golang-1.21.7-1.el9
```

Reply:
It won't last, as the repo won't carry this version forever. But other options are not workable either.